### ISSUE TITLE
Is any reduction in rate limit of 100 API calls per 5 minutes per IP.

### ISSUE THREAD

**Issue by conid4all** [cowinapi/developer.cowin#378]
I am seeing error "Too many search request , please try after sometime" and my search button is disabled despite I am searching 6 times per mint. This is happing from evening only.... 

Any idea how to solve this problem ....

Thanks

**Comment by throwawaylogical** [comment]
I can see in the XHR tab, that it does 2 API requests, one calendarBy and the other one /sessions/search API.
![image](https://user-images.githubusercontent.com/85205466/120410335-6b5de080-c370-11eb-9e61-2afbbf8c63f5.png)

 The /sessions/search API is rate limited to 20 hits per session, which means you will have to log out and log back in, while the calendarBy* API is rate limited to around 100 per 5 minutes per IP.

On the other hand, I've noticed that the /sessions/search API always ends up giving a 204 (Response: No Content), which means the data that is being populated (at least using a Desktop Browser) is being done by calendarBy* APIs.

When the button gets disabled, one can "Inspect Element" and remove the "button-disabled" property from the DOM element, and can search away freely. I'm hoping that this is a bug and that the /session/search APIs would be removed? It's highly infeasible to get a slot without constant searching in heavily populated areas.

**Comment by badinenisaivardhan** [comment]
Hey @conid4all, as per my experience if you are using public api end point they are limited to 100 per 5 mins, protected api endpoint for 20-30 searches

**Comment by aashimgarg** [comment]
hi @badinenisaivardhan  ccurrently the cowin api blocks any request originating outside of India. do u any free servers in india

**Comment by badinenisaivardhan** [comment]
hey @aashimgarg , well its true some public Api requests, and all protected Api request's are blocked out side the India, try to use a VPN server, but my suggestion is to go with any reputed company vpn it may cost you but its safe... try to avoid any free vpn there may a chance of data stealing...!!

**Comment by aashimgarg** [comment]
yes @badinenisaivardhan but other than vpn i think all the company servers are paid
